app/views/static/index.html.erb

```diff
@@ -0,0 +1,4 @@
+
+
+<h1>Static#index</h1>
+<p>Find me in app/views/static/index.html.erb</p>
```

Comment:
- [ ] According to the project requirement, to make your view file work with react, this file is meant to have only a `div`, with an id `root`. from your javascript/applicvation file where you initialize react dom, you can then get access to the div with the ID and then append react DOM to it.